tt_eager/tt_dnn/op_library/layernorm_distributed/kernels/dataflow/writer_unary_interleaved_start_id_blocked.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "dataflow_api.h"
+
+void kernel_main() {
```

Comment:
This kernel looks pretty basic; is there maybe another one we can use instead of writing a new one?